aries-site/src/pages/components/spinner.mdx

```diff
@@ -37,6 +37,9 @@ For additional clarity for screen reader users, provide a [message](https://v2.g
 
 [Message](https://v2.grommet.io/spinner#message) can be provided as a string or as an object in the format of `{ start: ... , end: ... }`, where the "start" message is announced when the Spinner appears and the "end" message is announced when the Spinner closes.
 
+### WCAG compliance
```

Comment:
For consistency with the other pages, can we move the entire Accessibility section to be the last section of the page?

In addition to moving the placement, this will require changing "### Accessibility" to "## Accessibility"